go.mod

```diff
@@ -57,6 +57,7 @@ require (
 	github.com/projectcalico/go-yaml v0.0.0-20161201183616-955bc3e451ef // indirect
 	github.com/projectcalico/go-yaml-wrapper v0.0.0-20161127220527-598e54215bee // indirect
 	github.com/projectcalico/libcalico-go v0.0.0-20190802144109-bf00054ed881
+	github.com/projectcalico/node v3.8.1+incompatible // indirect
```

Comment:
why did we pick up a dependency on node?

Reply:
Might have been leftover from an earlier change I made, I removed these and go didn't re-add them.